insights/links/low_reputation.yml

```
@@ -2,7 +2,7 @@ name: "Low reputation links"
 type: "query"
 source: |
     distinct(map(filter(body.links, 
-        .href_url.domain.root_domain not in $tranco_1m and .href_url.domain.valid !=false and .href_url.domain.root_domain not in $org_domains), 
+        .href_url.domain.root_domain not in $tranco_1m and .href_url.domain.valid !=false and .href_url.domain.root_domain not in $org_domains and .href_url.domain.root_domain not in ('sublime.security', 'sublimesecurity.com')), 
```

```suggestion
        .href_url.domain.root_domain not in $tranco_1m and .href_url.domain.valid !=false and .href_url.domain.root_domain not in $org_domains and .href_url.domain.root_domain not in $high_trust_sender_root_domains), 
```

swapping for high trust